azure-kusto-data/azure/kusto/data/_token_providers.py

```diff
@@ -470,7 +482,15 @@ def _get_token_from_cache_impl(self) -> dict:
 class InteractiveLoginTokenProvider(CloudInfoTokenProvider):
     """Acquire a token from MSAL with Device Login flow"""
 
-    def __init__(self, kusto_uri: str, authority_id: str, login_hint: Optional[str] = None, domain_hint: Optional[str] = None, is_async: bool = False):
+    def __init__(
+        self,
+        kusto_uri: str,
+        authority_id: str,
+        login_hint: Optional[str] = None,
+        domain_hint: Optional[str] = None,
+        is_async: bool = False,
+        proxy_dict: Optional[dict] = None,

```

Comment:
Inconsistency- get the proxy in constructor, but don't use it.
Check elsewhere as well for same issue.